src/codegen.ts

```diff
@@ -273,24 +273,33 @@ function queryValues(
 ////////////////////////////////////////////////////////////////////////
 
 export type TsModule = {
-  sqlFileName: string
-  tsFileName: string
+  sqlFileName: string // full path
+  tsFileName: string // full path
   funcName: string
 }
 
 export type TsModuleDir = {
-  dirPath: string
+  dirPath: string // full path
+  nestedDirs: TsModuleDir[]
   modules: TsModule[]
   hasErrors: boolean
 }
 
-export function generateIndexModule(modules: TsModule[]): string {
-  return modules
+export function generateIndexModule(
+  dirPath: string,
+  nestedDirs: TsModuleDir[],
+  modules: TsModule[]
+): string {
+  const nestedDirsStr = nestedDirs
+    .map((dir) => `export * from '${path.relative(dirPath, dir.dirPath)}';`)
```

Comment:
```suggestion
    .map((dir) => `export * from './${path.relative(dirPath, dir.dirPath)}';`)
```

Reply:
Without the leading `./`, you're trying to import a module from `node_modules` :)